### PR TITLE
Header Linter configuration

### PR DESCRIPTION
**Pull request by dinagraves**
Playing with the header linter.   Trying to make it fail and then pass.  

https://github.com/googleapis/repo-automation-bots/tree/master/packages/header-checker-lint#configuration

### CODE REVIEW

**Comment by dinagraves** [comment]
header-check successfully failed when missing a license - going to restore the header now

**Comment by dinagraves** [comment]
The check doesn't seem to be trigger on the file extensions from the new config, but that's probably cause it's not checked in yet.  It does work for the defaults: js, ts, java.

File: .bots/header-checker-lint.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ 	"ts", 
+ 	"js", 
+ 	"java", 
+ 	"sh", 
+ 	"Dockerfile", 
+ 	"yaml", 
+ 	"py",
+ 	"html",
+ 	"txt",
+ 	"json"
```
Comment (grayside) [inline .bots/header-checker-lint.json:14]: `.json` files are invalid with comments.

**Comment by dinagraves** [comment]
@grayside Yes, the bot is turned, but is only checking a few file types right now.

**Pull request merged**